docs/schemes/oauth2.md

```diff
@@ -27,7 +27,11 @@ auth: {
       redirect_uri: undefined,
       client_id: 'SET_ME',
       token_key: 'access_token',
-      state: 'UNIQUE_AND_NON_GUESSABLE'
+      state: 'UNIQUE_AND_NON_GUESSABLE',
+      code_challenge: '',
```

Comment:
@madmonkey you have a typo here, or? I think this should be - `code_challenge_method` - as least so I understand from your code and my test ;)

Reply:
You are absolutely right - I completely fudged the documentation.